src/mu_producer.cpp

```diff
@@ -39,7 +39,7 @@ bool MuonProducer::IsSignal(nano_tree &nano, int nano_idx, bool isZgamma) {
     if (fabs(eta) > MuonEtaCut) return false;
     if (pt > SignalMuonPtCut &&
       nano.Muon_miniPFRelIso_all()[nano_idx] < MuonMiniIsoCut &&
-      fabs(nano.Muon_dz()[nano_idx])<=0.5 &&
+      fabs(nano.Muon_dz()[nano_idx])<=0.5 && //Maybe compare with floats here. . .
```

Comment:
What question is this about? Maybe 0.5f would be consistent?

Reply:
I mainly wanted to avoid modifying code that would not be used in the zgamma analysis, but left a comment in there for other people. If it makes more sense to change everything, whether or not it is for a different analysis, I can do that.

Reply:
Oh. I didn't see the context of where this code was. Thanks for the clarification. I think it will be good to change it for the other code also, so that is is also 0.5f. Anyhow thanks for the consideration. It definitely helps!